var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -0,0 +1,69 @@
+from spack import *
+
+class Gdal(Package):
+    """
+    GDAL is a translator library for raster and vector geospatial
+    data formats that is released under an X/MIT style Open Source
+    license by the Open Source Geospatial Foundation. As a library,
+    it presents a single raster abstract data model and vector
+    abstract data model to the calling application for all supported
+    formats. It also comes with a variety of useful command line
+    utilities for data translation and processing
+    """
+
+    homepage   = "http://www.gdal.org/"
+    url        = "http://download.osgeo.org/gdal/2.0.2/gdal-2.0.2.tar.gz"
+    list_url   = "http://download.osgeo.org/gdal/"
+    list_depth = 2
+
+    version('2.0.2', '573865f3f59ba7b4f8f4cddf223b52a5')
+
+    extends('python')
```

Comment:
According to https://pypi.python.org/pypi/GDAL/, it can be built as a standalone Python package (py-gdal) that depends on the libgdal (gdal) library, or both can be built at the same time. Not sure how people feel about this. The tarballs themselves are different. But I don't think gdal needs to pull down anything different to build py-gdal like we ran into with NetCDF C/CXX/Fortran.